packages/lesswrong/server/manualMigrations/2024-06-05-rewriteOldReviewBotComments.ts

```diff
@@ -0,0 +1,51 @@
+import { Comments } from "@/lib/collections/comments";
+import { getSqlClientOrThrow } from "../sql/sqlClient";
+import { createAdminContext, updateMutator } from "../vulcan-lib";
+import { registerMigration } from "./migrationUtils";
+
+registerMigration({
+  name: "rewriteOldReviewBotComments",
+  dateWritten: "2024-06-05",
+  idempotent: true,
+  action: async () => {
+    const sql = getSqlClientOrThrow()
+
+    const context = createAdminContext()
+
+    const oldReviewBotComments: DbComment[] = await sql.many(`
+      SELECT * FROM "Comments"
+      WHERE "userId" = 'tBchiz3RM7rPwujrJ' AND "deleted" = FALSE
+    `);
+
+    const reviewBotUser: DbUser = await sql.one(`
+      SELECT * FROM "Users"
+      WHERE "_id" = 'tBchiz3RM7rPwujrJ'
+    `);
```

Comment:
For future reference, these can both just be `context.CollectionName.find(...)`, given they're single-table queries with basic WHERE clauses.  (`Comments.find({ userId: 'tBchiz3RM7rPwujrJ', deleted: false })` and `Users.findOne({ _id: 'tBchiz3RM7rPwujrJ' })`)

Reply:
:+1: